app/views/application/dashboard.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="container">
+  <div class="dash col-md-6">
+    <div class="dash">
+      <h1> daily dose </h1>
+      <span class="quote">
+        <%= @quotes.sample %>
+      </span>
+    </div>
+    <div class="dash">
+      <h1> sort by tag </h1>
+      <div class="tabs">
+        <% @tags.each do |tag| %>
+          <% if !params[:extra].nil? && params[:extra][:tag_ids] %>
+            <% if tag.id == params[:extra][:tag_ids].to_i %>
```

Comment:
i suggest a helper method for this as well. whenever you find yourself doing `&&` or `==` or `>=`, etc. then i consider that too much logic in your views. how can we refactor this?